nn_file_name: '2024-04-08_3joints_3.onnx'
nn_rate: 20
observation_size: 231
